Content/default/paket.lock

```diff
@@ -73,26 +73,26 @@ NUGET
       Fable.Browser.Dom (>= 2.4.4)
       Fable.Core (>= 3.2.7)
       FSharp.Core (>= 4.7.2)
-    Fable.Remoting.Client (7.13)
+    Fable.Remoting.Client (7.16)
       Fable.Browser.XMLHttpRequest (>= 1.0)
       Fable.Core (>= 3.1.5)
-      Fable.Remoting.MsgPack (>= 1.11)
-      Fable.SimpleJson (>= 3.20)
+      Fable.Remoting.MsgPack (>= 1.13)
+      Fable.SimpleJson (>= 3.21)
       FSharp.Core (>= 4.7)
-    Fable.Remoting.Giraffe (5.1)
-      Fable.Remoting.Server (>= 5.21)
+    Fable.Remoting.Giraffe (5.3)
+      Fable.Remoting.Server (>= 5.23)
       FSharp.Core (>= 5.0)
       Giraffe (>= 5.0)
-    Fable.Remoting.Json (2.16)
+    Fable.Remoting.Json (2.17)
```

Comment:
@Zaid-Ajaj This will be the proposed lock file for v3. I think from what you've said elsewhere, this is the correct version of Remoting?